src/Resources/public/assets/oney/3x4x.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="190px" height="40px" viewBox="0 0 190 40" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 59.1 (86144) - https://sketch.com -->
+    <title>ic_3x4x_oney_green</title>
```

Comment:
I suggest to pass this svg in a optimizator service to reduce size (see https://jakearchibald.github.io/svgomg/ for example :wink: )

Reply:
Done also on other branch. Let's merge this one and I'll do the merge on my branch :wink: